components/Tabs/Tabs.js

```diff
@@ -2,61 +2,68 @@
 class TabLink {
   constructor(element) {
     // Assign this.element to the passed in DOM element
-    // this.element;
+    this.element = element
     
     // Get the custom data attribute on the Link
-    // this.data;
+    this.data = this.element.querySelector(`.tabs-link[data-tab='${this.element.dataset.tab}']`);
```

Comment:
Another way to get the custom data attribute is by using the dataset.
*    this.data = this.element.dataset.tab;*